internal/datanode/importv2/util.go

```diff
@@ -52,6 +53,7 @@ func NewSyncTask(ctx context.Context,
 	segmentID, partitionID, collectionID int64, vchannel string,
 	insertData *storage.InsertData,
 	deleteData *storage.DeleteData,
+	bm25Stats *map[int64]*storage.BM25Stats,
```

Comment:
map of Go is a reference type, so we can just `bm25Stats map[int64]*storage.BM25Stats`